lib/ebay_api/operations/sell/inventory/inventory_item_groups/get.rb

```diff
@@ -0,0 +1,19 @@
+class EbayAPI
+  scope :sell do
+    scope :inventory do
+      scope :inventory_item_groups do
+        operation :get do
+          path { inventory_item_group_key }
+          option :inventory_item_group_key
+
+          http_method :get
+          query do
+            { inventoryItemGroupKey: inventory_item_group_key }
+              .select { |_, v| v }
+          end
+          response(404) { nil }
```

Comment:
Again, why do you want to just return `nil` on 404? 
Though I still not know a lot about this API method semantics, I suppose that if the application knows the `inventory_item_group_key` identifier it expects to retrieve something. Otherwise, it is a very exceptional thing that should be handled by application explicitly (and if not it is better for application to break with “I didn't found expected inventory item group” instead of “undefined method bar for nil”). But that is my opinion. Please share any arguments.

Reply:
I think you are right in this case. But I see that `nil` on 404 is [sometimes](https://github.com/nepalez/ebay_api/blob/95a76ef46213f3672a011b5f8b16243638a58f0d/lib/ebay_api/operations/sell/marketing/campaigns/get_by_name.rb#L15) returned. Maybe we should stick to Ruby on Rails convention (`find` versus `find_by`) and raise exceptions for `get` but return `nil` for `get_by` if not found.